content/en/load-testing/How-Tos/create-report-cheatsheet.md

```diff
@@ -0,0 +1,57 @@
+---
+title: "How to use Create_report Command Options (Cheat Sheet)"
+linkTitle: "Create_report Cheat Sheet"
+
+weight: 400
+type: docs
+
+description: >
+  How to make use of different create_report command options. <!-- You will find the usage instructions and a list of available commands below. -->
+---
+## Usage
+- `create_report [<options>] <inputdir>` is used to generate a test report.
+- `create_report --help` displays all the available command options and a brief guide about their usage.
+
+### Example
+
+#### A report can be generated in the following way, without using any options.
+##### Windows
+```powershell 
+cd <XLT>\bin 
+./create_report.cmd ../results/20210221-112110 
+```
+##### Linux
+```shell 
+cd <XLT>\bin 
+./create_report.sh ../results/20210221-112110 
+```
+
+## List of Command Options
+- `-ae --exclude-agents <agents>` can be used for [excluding a list of comma-separated agents.](../../manual/540-report-options/#report-for-a-subset-of-agents)
+
+- `-ai --include-agents <agents>` is to be used for [including a list of comma-separated agents.](../../manual/540-report-options/#report-for-a-subset-of-agents) 
+
+-  `-D <property=value>` will override a property from file.
+
+- `-e,--exclude-testcases <test cases>` [excludes a comma-separated list of test cases.](../../manual/540-report-options/#excluding-test-scenarios)
+
+- `-from <time>` can be used to ignore results generated before the given time. It can also be combined with `-l` and `-to` for [defining a reporting timeframe.](../../manual/540-report-options/#defining-a-reporting-timeframe)
+
+- `-i,--include-testcases <test cases>` can be used to [include a list of comma-separated test cases.](../../manual/540-report-options/#excluding-test-scenarios)
+- `-l <duration>` is used for utilizing results generated in the specified duration, must be used with `from` or `to` option.
```

Comment:
please link "specified duration" to #defining-a-reporting-timeframe (like with the `-from` parameter)